src/Language/PureScript/Errors.hs

```diff
@@ -189,6 +189,10 @@ nonEmpty = not . null . runMultipleErrors
 errorMessage :: SimpleErrorMessage -> MultipleErrors
 errorMessage err = MultipleErrors [ErrorMessage [] err]
 
+-- | Create an error set from a single simple error message and source annotation
+errorMessage' :: SourceSpan -> SimpleErrorMessage -> MultipleErrors
+errorMessage' ss err = MultipleErrors [ErrorMessage [] err]
```

Comment:
Yeah, that's one of the bits I hadn't finished up :) I left it bound but unused so the warning would occur.